services/kommander/0.1.0/dynamic-helmreleases/README.txt

```diff
@@ -0,0 +1,6 @@
+The purpose of these directories is to have ALL HelmRelease objects in the k-apps
+repo, including the ones that are generated dynamically.
```

Comment:
Can we add some explanation here on what "dynamically generated" HelmReleases are and why we need them, please? This will help our future selves recall what this dir is about.

Reply:
Comment added.